ajax/libs/chai-as-promised/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "chai-as-promised",
+  "version": "6.0.0",
+  "filename": "chai-as-promised.min.js",
+  "description": "Extends Chai with assertions about promises.",
+  "keywords": [
+    "chai",
+    "chai-plugin",
+    "browser",
+    "async",
+    "testing",
+    "assertions",
+    "promises",
+    "promises-aplus"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/domenic/chai-as-promised"
+  },
```

Comment:
I won't touch it here, it looks fine, not need to coming up with a standard way as there is no need.